test/e2e/utils/TestData.js

```diff
@@ -4,7 +4,7 @@
  */
 module.exports = {
   jurisdiction: 'Tribunals',
-  caseType: 'SSCS Case v5.0.36_AAT',
+  caseType: 'SSCS Case v5.0.65_AAT',
```

Comment:
Make this dynamic, hard coded will fail the tests once the definition file version changes

Reply:
Added in Create case pending search case file